Added MARC Coin metadata and logo in YAML format for Tonkeeper verification.

Ogamarcos opened the pull request:
address: 0:609351d7173a66f2716be1de541616152be2bb1640598d1ed7f414cb0b8612d1 symbol: MARC
name: MARC Coin
description: Marc Earn is a project dedicated to climate change and the planet revolution. Be part of this great journey today. decimals: 9
image: https://marccoin-bot.netlify.app/assets/marc-logo.png websites:
  - https://marccoinbot.netlify.app social:
  - https://t.me/marctapbot

Please make sure you change the original .yaml fields in the accounts/, collections/ or jettons/ directories and leave the auto-generated .json files in the repository root alone. Also please make sure that you do not use ton.api links in your pull request.
Example pull request:

```yaml
address: Address of your token 
symbol: Symbol of your token
websites:
  - "link"
social:
  - "link"
```

**ATTENTION! The Tonkeeper team does not charge any fees for checking/verifying tokens/collections. The procedure is ABSOLUTELY FREE. Ignore comments in commits that require payment to speed up/successfully check the PR. Stay safe fren!**

Пожалуйста, убедитесь, что вы изменили исходные поля .yaml в каталогах account/, Collections/ или jettons/ и не трогаете автоматически сгенерированные файлы .json в корне репозитория. Так же, пожалуйста, убедитесь, что вы не используете ссылки ton.api в вашем пул реквесте.
Пример пул реквеста:

```yaml
address: Адрес вашего токена 
symbol: Сивол вашего токена
websites:
  - "ссылка"
social:
  - "ссылка"
  ```

**ВНИМАНИЕ! Команда Tonkeeper не берет никакой оплаты за проверку/верификацию токенов/коллекций. Процедура АБСОЛЮТНО БЕСПЛАТНА. Игнорируйте комментарии в коммитах, требующих оплаты для ускорения/успешности проверки. Будьте внимательны и осторожны!**

Ogamarcos committed:
Create yaml

address: 0:609351d7173a66f2716be1de541616152be2bb1640598d1ed7f414cb0b8612d1
symbol: MARC
name: MARC Coin
description: Marc Earn is a project dedicated to climate change and the planet revolution. Be part of this great journey today.
decimals: 9
image: https://marccoin-bot.netlify.app/assets/marc-logo.png
websites:
  - https://marccoinbot.netlify.app
social:
  - https://t.me/marctapbot

Gene090 commented:
Hello. We recommend to develop the project further. 
Tonkeeper reserves the right to refuse token verification.

Ogamarcos commented:
> Hello. We recommend to develop the project further.
> Tonkeeper reserves the right to refuse token verification

Hello Tonkeeper Team,

I hope this message finds you well. Thank you for your feedback regarding the verification of our MARC Coin token.

Since your initial review, we have been actively working on enhancing the project. I’m excited to share that we have made several key improvements:
	1.	Added New Features: We’ve introduced [insert features like staking, rewards, or other utilities] to provide more value and utility for users holding MARC Coin.
	2.	Active Community Engagement: We have been actively engaging with our growing community on [Telegram/Discord/other platforms]. The feedback and support from our users have been fantastic, and we are continually working to foster a vibrant ecosystem around MARC Coin.
	3.	Expanded Roadmap: We have a clear roadmap for the future, with plans to [mention any future updates or developments you’re working on]. This includes expanding the use cases for MARC Coin and further integrating it within our platform.

We believe these enhancements will make MARC Coin an even more valuable part of the TON ecosystem. We kindly request that you reconsider our verification request. If there is any additional information or changes needed, we are more than happy to provide that.

Thank you for your time and consideration. We look forward to hearing from you and continuing to grow our project in collaboration with Tonkeeper.

Best regards,
[Maichiel dahouede]
MARC Coin Team
[+22997340859 Information]
